### PR TITLE
[PAN-2831]: Add info about filters being subscriptions

### PR DESCRIPTION
**Pull request by sgregglives**
## PR description

This fix adds content in several topics that clarifies the alternatives for subscriptions.

## Fixed Issue(s)

PAN-2831

### CODE REVIEW

File: docs/Privacy/Explanation/Privacy-Groups.md
Comment (NicolasMassart) [inline docs/Privacy/Explanation/Privacy-Groups.md:13]: It looks like PAN-2831 is not about privacy so this change me be a mistake in this PR.

**Review by NicolasMassart — CHANGES_REQUESTED** [review]
just an issue with aa change in Privacy-Groups.md that should not be there otherwise looks good to me. I will approve once the change done and merge conflict fixed.

File: docs/Pantheon-API/RPC-PubSub.md
Comment (MadelineMurray) [inline docs/Pantheon-API/RPC-PubSub.md:8]: Suggestion only - this could be rewritten more actively as: 
Subscribe to event using either ....

File: docs/Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md
Comment (MadelineMurray) [inline docs/Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md:3]: Similar comment to above: 
Subscribe to events, such as logs, using either filters over HTTP or RPC Pub/Sub....

**Review by MadelineMurray — APPROVED** [review]

**Comment by sgregglives** [comment]
I created another branch from master.

**Pull request closed without merging**
